lib/node_modules/@stdlib/stats/base/dists/logistic/cdf/benchmark/benchmark.js

```diff
@@ -33,16 +34,24 @@ var cdf = require( './../lib' );
 bench( pkg, function benchmark( b ) {
 	var mu;
 	var s;
+	var len;
 	var x;
 	var y;
 	var i;
 
+	len = 100;
+	mu = new Float64Array( len );
+	s = new Float64Array( len );
+	x = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( -100.0, 100.0 );
```

Comment:
```suggestion
		x[ i ] = uniform( -100.0, 0.0 );
```

Make sure you keep the same range as original.